Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -1929,8 +1937,8 @@ TextStream& operator<<(TextStream& ts, const FontCascade& fontCascade)
     ts << fontCascade.fontDescription();
 
     if (fontCascade.fonts()) {
-        ts << ", font selector " << fontCascade.fonts()->fontSelector();
-        ts << ", font selector version " << fontCascade.fonts()->fontSelectorVersion();
+        ts << ", font selector " << fontCascade.fontSelector();
+        ts << ", font selector version " << fontCascade.fontSelectorVersion();
```

Comment:
Shoud we move those out of the `if` ? They don't depend on it anymore

Reply:
yes, I think we should

Reply:
Also, operator<< for FontSelector already streams version, so we don't need to explicitly stream fontCascade.fontSelectorVersion